docs/service_x509_credentials_aws.md

```diff
@@ -0,0 +1,346 @@
+![AWS EC2 Setup](images/aws-ec2-setup.png)
+
+**Bootstrapping AWS EC2 instances with Athenz Service x.509 Identities
+requires the instances to have network connectivity to both Athenz ZTS
+and AWS STS Services.**
+
+## Domain Registration
+
+Identify your Athenz domain before you can proceed by using Athenz UI.
+
+You may create a top-level domain or a sub domain using Athenz UI or zms-cli.
+Only Athenz System Administrators can create top level domains (members of the sys.auth role).
```

Comment:
Replaced top level creation text. Original:
 Athenz top-level domain creation will be manual until we have Verizon Media
 product taxonomy like OPM to integrate with. If your product does not have
 a top level domain already registered in Athenz, you can file a JIRA
 ticket in the [ATHENS project](https://jira.ouroath.com/secure/CreateIssue.jspa?pid=10388&issuetype=10100).
 Please provide the following details:

 - a short and descriptive domain name for your product.
>     - Ex: video-data-platform
 - a short description about your product.
     - Ex: Data Pipeline for Video Analytics. part of Platform for Video Products (Atlas, Transcoding, ..)
 - list of administrators identified by their Verizon Media Okta Short IDs
     - Ex: mujibur, hga, mlakes
 - [Verizon Media Product Id](https://productmaster.vzbuilders.com/engineering/product)

 **Verizon Media Product Id is required for all Athenz Top Level Domains**

Reply:
I fixed all the other issues except this one. Not sure what you mean by generalizing. Please contact me when you have time.